CHANGELOG.md

```diff
@@ -255,3 +273,18 @@ Initial release.
 [0.3.0]: https://github.com/hyperledger-labs/go-perun/compare/v0.2.0...v0.3.0
 [0.2.0]: https://github.com/hyperledger-labs/go-perun/compare/v0.1.0...v0.2.0
 [0.1.0]: https://github.com/hyperledger-labs/go-perun/releases/v0.1.0
+
+[//]: # "Links for 0.8.0"
+[#225]: https://github.com/hyperledger-labs/go-perun/pull/225
+[#227]: https://github.com/hyperledger-labs/go-perun/pull/227
+[#228]: https://github.com/hyperledger-labs/go-perun/pull/228
+[#124]: https://github.com/hyperledger-labs/go-perun/pull/124
+[#144]: https://github.com/hyperledger-labs/go-perun/pull/144
+[#204]: https://github.com/hyperledger-labs/go-perun/pull/204
+[#129]: https://github.com/hyperledger-labs/go-perun/pull/129
+[#134]: https://github.com/hyperledger-labs/go-perun/pull/134
+[#148]: https://github.com/hyperledger-labs/go-perun/pull/148
+[#191]: https://github.com/hyperledger-labs/go-perun/pull/191
+[#196]: https://github.com/hyperledger-labs/go-perun/pull/196
+[#207]: https://github.com/hyperledger-labs/go-perun/pull/207
+[#218]: https://github.com/hyperledger-labs/go-perun/pull/218
```

Comment:
`#123`'s are automatically translated by Github, no need for this section (which would grow unnecessarily long if we followed this pattern in the future). You can just remove the `[...]` around them in above text. See e.g. [geth release notes](https://github.com/ethereum/go-ethereum/releases/tag/v1.10.12), they also just mention the issue/PR numbers as `#123`.

Reply:
This is not the case, see [here](https://github.com/perun-network/go-perun/blob/f57bff55dcf119a85cd0f5766ebf681b8078240d/CHANGELOG.md).

Reply:
Sorry, you're totally right. Don't know why I thought that...
I still propose that we inline those links and not put them at the end of the document as footnotes.

Reply:
I definitely agree with your point that this is somehow polluting the CHANGELOG. Open for suggestions. One alternative of course is to not link the issues.

Reply:
Hmm I thought that this is somehow visible in the final CHANGELOG, but it isn't. Maybe we move this list of links always at the end of each changelog entry (so above the title of the next changelog entry)? Then it doesn't grow into a very long list and stays together with the entry.